src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala

```diff
@@ -40,6 +43,254 @@ class TimeToPayProxySteps extends ScalaDsl with EN with Eventually with Matchers
     TimeToPayProxyRequests.createPlanRequestBody(dataTable)
   }
 
+  Given("a create plan request with an empty customer reference") { () =>
+    val request = CreatePlanRequest(
+      "",
+      "QuoteRef123",
+      "advisor",
+      Plan(
+        "QuoteId1",
+        "instalmentAmount",
+        LocalDate.now().toString,
+        LocalDate.now().plusDays(5),
+        100,
+        "instalmentOrder",
+        false,
+        10,
+        "monthly",
+        10,
+        LocalDate.now().plusDays(5),
+        100,
+        10000,
+        500,
+        100,
+        50
+      ),
+      Seq(DebtItemCharge("Debt1", "1525", "1000", 500, LocalDate.of(2019, 10, 5), Some(Seq(ChargePayment(LocalDate.now().minusWeeks(5), 100))))),
+      Seq(Payment("BACS", "PRef1")),
+      Seq(PostCode("NW9 5XW", LocalDate.now().minusMonths(6))),
+      Seq(Instalment("Debt1", LocalDate.now().plusMonths(1), 500, 500, 0.26, 1, 50, 10000 - 500))
+    )
+    ScenarioContext.set("createPlanRequest", Json.toJson(request).toString())
+  }
+
+  Given("a create plan request with an empty quote reference") { () =>
+    val request = CreatePlanRequest(
+      "CustomerRef123",
+      "",
+      "advisor",
+      Plan(
+        "QuoteId1",
+        "instalmentAmount",
+        LocalDate.now().toString,
+        LocalDate.now().plusDays(5),
+        100,
+        "instalmentOrder",
+        false,
+        10,
+        "monthly",
+        10,
+        LocalDate.now().plusDays(5),
+        100,
+        10000,
+        500,
+        100,
+        50
+      ),
+      Seq(DebtItemCharge("Debt1", "1525", "1000", 500, LocalDate.of(2019, 10, 5), Some(Seq(ChargePayment(LocalDate.now().minusWeeks(5), 100))))),
+      Seq(Payment("BACS", "PRef1")),
+      Seq(PostCode("NW9 5XW", LocalDate.now().minusMonths(6))),
+      Seq(Instalment("Debt1", LocalDate.now().plusMonths(1), 500, 500, 0.26, 1, 50, 10000 - 500))
+    )
+    ScenarioContext.set("createPlanRequest", Json.toJson(request).toString())
+  }
+
+  Given("a create plan request with an empty channel identifier") { () =>
```

Comment:
Already covered by 'Scenario: TTP Promote an Arrangement request to create a plan for storage -empty channelIdentifier'?

Reply:
Yeah - this step is to provide data for that scenario